core/controllers/domain_objects_validator.py

```diff
@@ -243,3 +243,49 @@ def validate_suggestion_images(files):
     # The files argument do not represent any domain class, hence dict form
     # of the data is returned from here.
     return files
+
+
+def validate_exploration_change(exp_change_dict):
+    """Validate the exploration change list
+
+    Args:
+        exp_change_dict: dict. The exploration change dictionary
+            that needs to be validated.
+
+    Returns:
+        exp_domain.ExplorationChange. The exploration change domain
```

Comment:
It's not clear to me why this is being returned, given the name of the function. It sounds like the function takes a dict and validates it; why is it also converting the dict to a domain object in that case?

Reply:
Yeah, For validating the object, previously we were using `object_class` and in the schema utils file it was simply getting converted to the domain object and then pass on further. When I added the `validation_method` key for further validation, I have to return the domain object from validation function as `schema_utils` file simply pass on the result that it gets from the validation function. The reason why it needs to get converted to domain object is because all the further implementation of this handler is based on the domain object not the dictionary. I will update the function name.
Thanks for pointing it out
Hitesh